libraries/AP_Motors/AP_MotorsMatrix.cpp

```diff
@@ -1057,6 +1057,24 @@ void AP_MotorsMatrix::setup_motors(motor_frame_class frame_class, motor_frame_ty
                             };
                     add_motors(motors, ARRAY_SIZE(motors));
                     break;
+                }
+                // BF/X cinelifters using two 4-in-1 ESCs are quite common
+                // see: https://fpvfrenzy.com/betaflight-motor-order/
+                case MOTOR_FRAME_TYPE_BF_X: {
+                    _frame_type_string = "BF_X";
+                    static const AP_MotorsMatrix::MotorDef motors[] {
+                        {  135, AP_MOTORS_MATRIX_YAW_FACTOR_CW,  2 },
+                        {   45, AP_MOTORS_MATRIX_YAW_FACTOR_CCW, 1 },
+                        { -135, AP_MOTORS_MATRIX_YAW_FACTOR_CCW, 3 },
+                        {  -45, AP_MOTORS_MATRIX_YAW_FACTOR_CW,  4 },
+                        {  135, AP_MOTORS_MATRIX_YAW_FACTOR_CCW, 6 },
+                        {   45, AP_MOTORS_MATRIX_YAW_FACTOR_CW,  5 },
+                        { -135, AP_MOTORS_MATRIX_YAW_FACTOR_CW,  7 },
+                        {  -45, AP_MOTORS_MATRIX_YAW_FACTOR_CCW, 8 },
```

Comment:
```suggestion
                        {  135, AP_MOTORS_MATRIX_YAW_FACTOR_CW,  3 },
                        {   45, AP_MOTORS_MATRIX_YAW_FACTOR_CCW, 1 },
                        { -135, AP_MOTORS_MATRIX_YAW_FACTOR_CCW, 5 },
                        {  -45, AP_MOTORS_MATRIX_YAW_FACTOR_CW,  7 },
                        {  135, AP_MOTORS_MATRIX_YAW_FACTOR_CCW, 4 },
                        {   45, AP_MOTORS_MATRIX_YAW_FACTOR_CW,  2 },
                        { -135, AP_MOTORS_MATRIX_YAW_FACTOR_CW,  6 },
                        {  -45, AP_MOTORS_MATRIX_YAW_FACTOR_CCW, 8 },
```
I think the motor test order should be this. This is the diagram were trying to match:
![image](https://user-images.githubusercontent.com/33176108/181214271-367ef238-2fb5-4af6-a5cc-4cc23a692436.png)